Sources/NIOCore/ByteBuffer-hex.swift

```diff
@@ -341,3 +355,54 @@ extension ByteBuffer {
         }
     }
 }
+
+enum HexDecodingError: Error {
+    case invalidHexLength
+    case invalidCharacter
+}
+
+extension UInt8 {
+    fileprivate var isASCIIWhitespace: Bool {
+        [UInt8(ascii: "\n"), UInt8(ascii: "\t"), UInt8(ascii: "\n"), UInt8(ascii: "\r"), UInt8(ascii: " ")].contains(
```

Comment:
We don't need the duplicate.

```suggestion
        [UInt8(ascii: "\t"), UInt8(ascii: "\n"), UInt8(ascii: "\r"), UInt8(ascii: " ")].contains(
```